core/templates/dev/head/domain/collection/EditableCollectionBackendApiService.js

```diff
@@ -30,21 +30,18 @@
 // it if it's agreed upon).
 oppia.factory('EditableCollectionBackendApiService', [
     '$http', '$q', 'COLLECTION_DATA_URL_TEMPLATE',
-    'COLLECTION_EDITABLE_DATA_URL_TEMPLATE', 'UrlInterpolationService',
+    'EDITABLE_COLLECTION_DATA_URL_TEMPLATE', 'UrlInterpolationService',
     'ReadOnlyCollectionBackendApiService',
     function($http, $q, COLLECTION_DATA_URL_TEMPLATE,
-      COLLECTION_EDITABLE_DATA_URL_TEMPLATE, UrlInterpolationService,
+      EDITABLE_COLLECTION_DATA_URL_TEMPLATE, UrlInterpolationService,
       ReadOnlyCollectionBackendApiService) {
       var _fetchCollection = function(
           collectionId, successCallback, errorCallback) {
         var collectionDataUrl = UrlInterpolationService.interpolateUrl(
-          COLLECTION_DATA_URL_TEMPLATE, {
+          EDITABLE_COLLECTION_DATA_URL_TEMPLATE, {
             collection_id: collectionId
           });
 
-        // Does not work with following line
-        // collectionDataUrl += '?allow_invalid_explorations=true';
```

Comment:
Removed this switch as the read_only and editable controller handles allow_invalid_explorations implicitly.